Alkahest.Core/Net/ServerListRequestHandler.cs

```diff
@@ -68,8 +68,9 @@ string GetAndAdjustServers(out IReadOnlyList<ServerInfo> servers)
 
             var doc = XDocument.Parse(resp.Content.ReadAsStringAsync().Result);
             var servs = new List<ServerInfo>();
+            var elems = _parameters.Region == Region.JP ? doc.Root.Elements("serverlist").Elements("server") : doc.Root.Elements("server");
 
-            foreach (var elem in doc.Root.Elements("server"))
```

Comment:
I think we can simply make this `doc.Root.Descendants("server")` to deal with the difference in JP's SLS. That way, we shouldn't need the conditional above.